test/service.go

```diff
@@ -103,3 +186,25 @@ func UpdateServiceRevisionTemplateMetadata(logger *logging.BaseLogger, clients *
 	}
 	return clients.ServingClient.Services.Patch(svc.ObjectMeta.Name, types.JSONPatchType, patchBytes, "")
 }
+
+// GetServiceLatestRevision takes a revision in through names and compares it to the current state of LatestCreatedRevisionName in Service.
+// Once an update is detected in the LatestCreatedRevisionName, the function waits for the created revision to be set in LatestReadyRevisionName
+// before returning the name of the revision.
+func GetServiceLatestRevision(clients *Clients, names ResourceNames) (string, error) {
```

Comment:
Same comment as `GetConfigLatestRevision` but feel free to ignore

Reply:
Changing to WaitForConfigLatestRevision